vms/avm/txs/mempool/mempool.go

```diff
@@ -44,13 +44,16 @@ var (
 // Mempool contains transactions that have not yet been put into a block.
 type Mempool interface {
 	Add(tx *txs.Tx) error
-	Has(txID ids.ID) bool
-	Get(txID ids.ID) *txs.Tx
```

Comment:
Generally I think having `Has` and `Get` on the mempool isn't very async friendly because the mempool is now expected to change concurrently with interactions with it.